src/components/share/ShareDirective.js

```diff
@@ -47,6 +47,19 @@ goog.require('ga_permalink');
         }
         // Use clipboard API to copy URL in OS clipboard
         element.on('click', function() {
+
+            // the copy link for the iframe-preview is refreshed
+            // according to the preview changes
+            var previewFrame;
+            [previewFrame] = angular.element(document).find('iframe');
+            if (previewFrame) {
+              var url = previewFrame.contentWindow.location.href;
+              var frame = $('.ga-embed-input')[1].value;
```

Comment:
don't need to use ` .ga-embed-input `. Use the inputToCopy variable declared further.

```
var inputToCopy = $(attrs.gaShareCopyBt);
var previewFrame;
[previewFrame] = angular.element(document).find('iframe');
if (previewFrame) {
    var url = previewFrame.contentWindow.location.href;
    var frame = inputToCopy.val();
.
.
.
.


```